syft/pkg/cataloger/binary/test-fixtures/classifiers/positive/argocd-2.5.11/README.md

```diff
@@ -0,0 +1,39 @@
+test-fixtures for https://github.com/anchore/syft/issues/1606
```

Comment:
Thanks for including a README.md about the test fixtures. 

My suggestion here is to point you towards a recent PR that was merged regarding the Ruby classifier:
https://github.com/anchore/syft/pull/1665/files

Rather than have the README that can be built, can we incorporate the test fixtures in the same pattern used in that PR as seen in the `Makefile`? This would hook the testing of these specific binaries into the same pattern that we're using for python and ruby

Reply:
Thanks for your recommendation. Shifting the `README.md` which deals with test fixtures into the `Makefile` is a superior choice.